home-work-app/src/app/redux/store.ts

```diff
@@ -3,6 +3,9 @@ import thunk from 'redux-thunk';
 import { movieEditor } from './reducers/movieEditor';
 
 // export const store = createStore(combineReducers(reducers), applyMiddleware(thunk));
-export const store: Store = createStore(combineReducers({
-    movieEditor
-}), applyMiddleware(thunk));
\ No newline at end of file
+export const store: Store = createStore(
+    combineReducers({
+        movieEditor,
+    }),
+    applyMiddleware(thunk)
```

Comment:
Add here extention for `redux_dev_tool`, this extention is indispensable when you are using redux.